test/specs/callApi.spec.js

```diff
@@ -1,13 +1,31 @@
+import fetchMock from 'fetch-mock'
+import { startsWith } from 'lodash/fp'
+
 import expect from '../expect'
 
 import callApi from '../../src/callApi'
 
 describe('callApi', () => {
+  afterEach(() => {
+    fetchMock.restore()
+  })
   it('should return error when no absolute url provided', async () => {
     const response = await callApi('')
 
     expect(response).to.eql({
       error: 'Error parsing the response: only absolute urls are supported',
     })
   })
+
+  it('should return response with result, responseHeaders and entities', async () => {
+    fetchMock.get(startsWith('/comments'), {
+      body: [{ id: '1', body: 'My first comment' }],
+    })
+
+    const { response } = await callApi('/comments', {}, {})
+
+    expect(response).to.have.property('result')
```

Comment:
It would be great, if you'd actually also check that these properties have the correct representations and not only that they exist.